src/Elmish.WPF/ViewModel.fs

```diff
@@ -211,13 +211,21 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     | OneWaySeqLazyData d ->
         let get = measure name "get" d.Get
         let map = measure name "map" d.Map
+        let getId = measure name "getId" d.GetId
+        let values = ObservableCollection(initialModel |> get |> map)
+        let valuesById = Dictionary<_,_>(values.Count)
+        for value in values do
+          let id = getId value
+          if valuesById.ContainsKey id
+          then failwithf "The getId function is supposed to return distinct IDs for each element, but it returned the same ID %A for the two elements %A and %A" id (valuesById.[id]) value
```

Comment:
I suggest changing `is supposed to` to `must` in all messages.

Reply:
done